documentation/src/docs/asciidoc/release-notes-5.0.0-M4.adoc

```diff
@@ -34,6 +34,8 @@ on GitHub.
 * `--hide-details` option of the `ConsoleLauncher` is deprecated, use `--details none` instead.
 * The ZIP distribution containing the console launcher is no longer provided. It is replaced
   by an executable standalone JAR distribution. For details, see "New Features" section below.
+* ExtensionContext composite Namespaces with the same parts but in different order will
+  not be considered equal.
```

Comment:
Hi @gaganis, I think this would read better if it looked something like:

> `ExtensionContext.Namespace`s with the same parts but in a different order are no longer considered equal.